harmonica/ellipsoid.py

```diff
@@ -140,6 +142,11 @@ def second_eccentricity(self):
         "The second eccentricity [adimensional]"
         return self.linear_eccentricity / self.semiminor_axis
 
+    @property
+    def mean_radius(self):
+        "The arithmetic mean radius [meters]"
```

Comment:
Would be good to add the definition and reference to Mortiz in the docstring. This is rendered in the class documentation.